tests/test_routes.py

```diff
@@ -141,6 +141,337 @@ def test_cves_returns_422_for_non_existing_package_status(self):
         assert response.status_code == 422
         assert "Cannot find a status" in response.json["errors"]
 
+    def test_cve_updated_at_column(self):
+        """
+        Tests that updated_at is created automatically when a new CVE is added
+        and that it is changed on update.
+        """
+
+        cve_payload = payloads.cve1.copy()
+
+        add_cve_response = self.client.put(
+            "/security/cves.json",
+            json=[cve_payload],
+        )
+
+        assert add_cve_response.status_code == 200
+
+        cve = self.client.get(f"/security/cves/{cve_payload['id']}.json").json
+
+        # Check that value exists and is populated
+        assert cve["updated_at"] is not None
```

Comment:
I think in the case where the CVE has just been created, we should expect `updated_at` to be exactly equal to `published`, and so it would be good to test for that.